ops/ccs-ops-misc/load_test/v1/baseline_tests.py

```diff
@@ -0,0 +1,131 @@
+import random
```

Comment:
Maybe we want to call this the regression suite or something instead of baseline tests? The baseline will be used to validate them, but they are intended to be used as regression tests. (same with the v2 name)

Reply:
Yeah, I think that's a better name for these suites. I'll rename them to the following:

* `v1/baseline_tests.py` -> `v2/regression_suite.py`
* `v2/baseline_tests.py` -> `v2/regression_suite.py`